src/Adapter/Order/Refund/OrderRefundCalculator.php

```diff
@@ -114,44 +128,64 @@ private function getOrderTailList(array $orderDetailRefunds)
      * @param array $orderDetailRefunds
      * @param bool $isTaxIncluded
      * @param array $orderDetails
+     * @param TaxCalculator $taxCalculator
+     * @param int $precision
      *
      * @return array
      *
      * @throws PrestaShopDatabaseException
      * @throws PrestaShopException
      */
-    private function flattenProductRefunds(array $orderDetailRefunds, bool $isTaxIncluded, array $orderDetails)
+    private function flattenCheckedProductRefunds(array $orderDetailRefunds, bool $isTaxIncluded, array $orderDetails, TaxCalculator $taxCalculator, int $precision)
     {
         $productRefunds = [];
         /** @var OrderDetailRefund $orderDetailRefund */
         foreach ($orderDetailRefunds as $orderDetailRefund) {
             $orderDetailId = $orderDetailRefund->getOrderDetailId();
+            /** @var OrderDetail $orderDetail */
             $orderDetail = $orderDetails[$orderDetailId];
             $quantity = $orderDetailRefund->getProductQuantity();
+            $quantityLeft = (int) $orderDetail->product_quantity - (int) $orderDetail->product_quantity_refunded - (int) $orderDetail->product_quantity_return;
+            if ($quantity > $quantityLeft) {
+                throw new InvalidRefundQuantityException(InvalidRefundQuantityException::QUANTITY_TOO_HIGH, $quantityLeft);
```

Comment:
```suggestion
                throw new InvalidRefundQuantityException(
                    InvalidRefundQuantityException::QUANTITY_TOO_HIGH, 
                    $quantityLeft
                );
```

Reply:
I believe cs-fixer changed this for me 😅 But I'll try again

Reply:
Weird, fuck php-cs-fixer sometimes :/

Reply:
Yep I confirm cs-fix changes it back to one line, I won't search for hours why for now ^^

Reply:
FUCK cs fixer again